geoportal/c2cgeoportal_geoportal/scaffolds/update/+dot+upgrade.yaml_tmpl

```diff
@@ -25,15 +28,19 @@ default_project_file:
     - \.circleci/config\.yml
     - \.travis\.yml
     - qgisserver/geomapfish\.yaml
-    - build
   exclude:
+    - geoportal/{{package}}_geoportal/static/apihelp
     - geoportal/{{package}}_geoportal/static-ngeo/js/{{package}}module\.js
     - mapserver/demo\.map\.tmpl
 
+# Files that we dont probide any diff => will just be keeped as is.
```

Comment:
Files ignored when creating the diff file => will just be untouched. (or kept as is)